recipes/yt/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py --quiet install --single-version-externally-managed --record=record.txt
```

Comment:
Is this really necessary? Since the windows build line is slightly different I'd prefer to keep them as separate files.